app/util/date.ts

```diff
@@ -1,10 +1,9 @@
-/* eslint-disable import/prefer-default-export */
 /**
  * Converts a Date object into an ISO String representation (truncates milliseconds)
  *
  * @param date - The date to convert
  * @returns An ISO string representation of the date, with milliseconds truncated
  */
-export function toISODateTime(date: Date): string {
+export default function toISODateTime(date: Date): string {
```

Comment:
Noted above, I probably just disagree with the linter rule in this case.  It makes sense for this to be the default export only if we don't plan to have other date utility methods.

Reply:
Fixed.